packages/clients/diplan/src/plugins/Gfi/Content.vue

```diff
@@ -1,13 +1,217 @@
 <template>
-  <v-card class="diplan-gfi-content"> TODO: NOT IMPLEMENTED </v-card>
+  <v-card class="diplan-gfi-content">
+    <v-card-actions>
+      <div>
+        <v-card-title class="py-0 pb-4">BPlan Vektor</v-card-title>
+        <v-card-subtitle class="py-0">1 Layer</v-card-subtitle>
+      </div>
+      <v-spacer></v-spacer>
+      <v-btn
+        icon
+        small
+        :aria-label="$t('plugins.gfi.header.close')"
+        @click="close(true)"
+      >
+        <v-icon small>fa-xmark</v-icon>
+      </v-btn>
+    </v-card-actions>
+    <v-divider class="mx-4" />
+    <v-card-actions class="mx-4">
+      <v-card-subtitle>
+        {{ featureItems.length }} WMS-Features
+        <v-tooltip bottom>
+          <template #activator="{ on }">
+            <v-icon small class="mb-1" v-on="on">fa-info-circle</v-icon>
+          </template>
+          <span
+            >Die Anzahl der aufgeführten WMS-Features ist abhängig vom
+            Klickradius und der Zoomstufe.</span
+          >
+        </v-tooltip>
+      </v-card-subtitle>
+      <v-divider vertical class="mx-2"></v-divider>
+      <v-select
+        v-model="feature"
+        :items="featureItems"
+        return-object
+        dense
+        hide-details
+        label="Feature"
+      ></v-select>
+    </v-card-actions>
+    <v-tabs v-model="tab" class="mx-4">
+      <v-tab
+        v-for="table of Object.keys(tables)"
+        :key="table"
+        class="text-none"
+        >{{ table }}</v-tab
+      >
+    </v-tabs>
+    <v-tabs-items v-model="tab" class="mx-4">
+      <v-tab-item v-for="[key, filter] of Object.entries(tables)" :key="key">
+        <v-simple-table v-if="getProperties(filter).length" dense>
+          <template #default>
+            <tbody>
+              <tr
+                v-for="{ label, value } of getProperties(filter)"
+                :key="key + label"
+              >
+                <th>{{ label }}</th>
+                <td>{{ value }}</td>
+              </tr>
+            </tbody>
+          </template>
+        </v-simple-table>
+        <b v-else>Keine Attribute vorhanden!</b>
+      </v-tab-item>
+    </v-tabs-items>
+    <div class="mt-4 d-flex">
+      <v-btn
+        style="flex: 1"
+        :disabled="featureIndex <= 0"
+        :aria-disabled="featureIndex <= 0"
+        aria-label="Vorheriges Feature aufrufen"
+        @click="navigate(-1)"
+      >
+        <v-icon small>fa-chevron-left</v-icon>
+      </v-btn>
+      <v-btn
+        style="flex: 1"
+        :disabled="featureIndex >= featureItems.length - 1 || featureIndex < 0"
+        :aria-disabled="
+          featureIndex >= featureItems.length - 1 || featureIndex < 0
+        "
+        aria-label="Nächstes Feature aufrufen"
+        @click="navigate(1)"
+      >
+        <v-icon small>fa-chevron-right</v-icon>
```

Comment:
```suggestion
        <v-icon color="black">$vuetify.icons.chevron-left</v-icon>
```
Yup, it's reversed in this icon set ^^